doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1225,6 +1225,12 @@ Other libraries
 
   * Added the :c:func:`date_time_now_local` function to the API.
 
+* :ref:`dk_buttons_and_leds_readme` library:
+
+  * Added an experimental no interrupts mode for button handling.
+    You can enable it with the :kconfig:option:`CONFIG_DK_LIBRARY_BUTTON_NO_ISR` Kconfig option.
+    It is a workaround that can be used to avoid using the GPIO interrupts, however, it increases the power consumption.
```

Comment:
```suggestion
    To enable this mode, use the :kconfig:option:`CONFIG_DK_LIBRARY_BUTTON_NO_ISR` Kconfig option.
    You can use this mode as a workaround to avoid using the GPIO interrupts.
    However, using it increases the power consumption.
```

Reply:
Can we change from "using" to e.g., 'However, doing so increases...'  (?) as this edit causes the word to repeat 3 times within 2 sentences

Reply:
You are right, I noticed this. Maybe just drop the word "using" from the last sentence.

Reply:
Agreed! So let's go back to the previous version but keep it split as per suggestion into 2 sentences.